static/js/services/search.js

```diff
@@ -34,7 +34,8 @@ var _ = require('underscore');
         return false;
       };
 
-      var getPage = function(type, rows, options) {
+      // Get the subset of rows, in appropriate order, according to options.
+      var getPageSubset = function(type, rows, options) {
```

Comment:
Hmm I get that. 
`getPageRows`? My first thought for `page` is webpage, the whole thing with html and stuff, that's why I'm tweaking.